### PR TITLE
Feature/offer get insured section

### PR DESCRIPTION
**Pull request by frapperino**

### CODE REVIEW

File: src/pages/Offer/sections/GetInsured.tsx [hunk 1]
```diff
@@ -83,7 +130,7 @@ export const GetInsured: React.SFC<Props> = ({
             {() => (
               <TranslationsConsumer textKey="OFFER_SUMMARY_SIGN_CTA">
                 {(ctaText) => (
-                  <GetInsuredButton>
+                  <GetInsuredButton style={{ marginTop: '30px' }}>
```
Comment (palmfjord) [inline src/pages/Offer/sections/GetInsured.tsx]: inline styling :/

Comment (frapperino) [reply src/pages/Offer/sections/GetInsured.tsx]: I know, since the button is global:/ Any nicer way to do that?

Comment (palmfjord) [reply src/pages/Offer/sections/GetInsured.tsx]: you can pass props to the button component and pick it up in the styled component :) like this https://github.com/HedvigInsurance/web-onboarding/blob/master/src/components/buttons.tsx#L21

File: src/pages/Download/sections/DownloadApp.tsx [hunk 2]
```diff
@@ -92,7 +92,7 @@ export const DownloadApp: React.SFC = () => (
           {(insuredText) => insuredText}
         </TranslationsConsumer>
       </InsuredText>
-      <GetInsuredButton>
+      <GetInsuredButton margin="0px">
```
Comment (palmfjord) [inline src/pages/Download/sections/DownloadApp.tsx]: we could just set a default to `0` and make it optional :D

File: src/components/buttons.tsx [hunk 3]
```diff
@@ -3,11 +3,11 @@ import styled from 'react-emotion'
 import { Link } from 'react-router-dom'
 
 export const GetInsuredButton = styled('div')(
-  ({ margin }: { margin: string }) => ({
+  ({ margin }: { margin?: string }) => ({
     display: 'flex',
     flexDirection: 'row',
     justifyContent: 'center',
-    marginTop: margin,
+    marginTop: margin ? margin : 0,
```
Comment (palmfjord) [inline src/components/buttons.tsx]: sorry to complain so much but `margin || 0` would be sufficient really 😅

Comment (hedvigoscar) [reply src/components/buttons.tsx]:
```suggestion
    marginTop: margin || 0,
```

**Pull request merged**